### PR TITLE
sarmiento add view list offense

### PR DESCRIPTION
**Pull request by Vince-1221**

### CODE REVIEW

File: prefect-office-record-management-system/src/main/java/com/prefect/office/record/management/app/facade/prefect/offense/impl/OffenseFacadeImpl.java [hunk 1]
```diff
@@ -5,13 +5,23 @@
...
+import java.util.List;
+
 public class OffenseFacadeImpl implements OffenseFacade {
 
     OffenseDao offenseDao = new OffenseDaoImpl();
 
     public OffenseFacadeImpl(OffenseDao offenseDao){
 
     }
+    public List<Offense> getAllOffenses() {
```
Comment (RoCS2024) [inline prefect-office-record-management-system/src/main/java/com/prefect/office/record/management/app/facade/prefect/offense/impl/OffenseFacadeImpl.java:17]: why return null?

File: prefect-office-record-management-system/src/main/java/com/prefect/office/record/management/app/facade/prefect/offense/impl/OffenseFacadeImpl.java [hunk 2]
```diff
@@ -5,13 +5,23 @@
...
+        return null;
+    }
+
+    @Override
+    public <Student> Student getStudentById(String id) {
```
Comment (RoCS2024) [inline prefect-office-record-management-system/src/main/java/com/prefect/office/record/management/app/facade/prefect/offense/impl/OffenseFacadeImpl.java:22]: why return null?

**Review by RoCS2024 — CHANGES_REQUESTED** [review]
check comments

**Review by RoCS2024 — CHANGES_REQUESTED** [review]
check comments

File: offense-view-application/src/main/java/com/offense/view/app/facade/offense/impl/OffenseFacadeImpl.java [hunk 3]
```diff
@@ -28,6 +28,4 @@ public Offense getOffenseById(String offenseId) {
     public <Student> Student getStudentById(String id) {
         return null;
```
Comment (RoCS2024) [inline offense-view-application/src/main/java/com/offense/view/app/facade/offense/impl/OffenseFacadeImpl.java:29]: why is this still returning null?

File: offense-view-application/src/main/java/com/offense/view/app/facade/offense/impl/OffenseFacadeImpl.java [hunk 4]
```diff
@@ -28,6 +28,4 @@ public Offense getOffenseById(String offenseId) {
     public <Student> Student getStudentById(String id) {
```
Comment (RoCS2024) [inline offense-view-application/src/main/java/com/offense/view/app/facade/offense/impl/OffenseFacadeImpl.java:28]: why still returning null?

File: prefect-office-record-management-system/src/main/java/com/prefect/office/record/management/app/facade/prefect/offense/impl/OffenseFacadeImpl.java [hunk 5]
```diff
@@ -5,13 +5,19 @@
...
 import com.prefect.office.record.management.data.dao.prefect.offense.impl.OffenseDaoImpl;
 
+import java.util.ArrayList;
+import java.util.List;
+
 public class OffenseFacadeImpl implements OffenseFacade {
 
     OffenseDao offenseDao = new OffenseDaoImpl();
 
     public OffenseFacadeImpl(OffenseDao offenseDao){
```
Comment (RoCS2024) [inline prefect-office-record-management-system/src/main/java/com/prefect/office/record/management/app/facade/prefect/offense/impl/OffenseFacadeImpl.java:15]: why is this necessary?

**Review by RoCS2024 — CHANGES_REQUESTED** [review]
see comments

File: offense-view-application/src/main/java/com/offense/view/app/facade/offense/impl/OffenseFacadeImpl.java (hunk 4 above)
Comment (Vince-1221) [reply offense-view-application/src/main/java/com/offense/view/app/facade/offense/impl/OffenseFacadeImpl.java:28]: i rely on what my scrum master expert said lang po sir

File: prefect-office-record-management-system/src/main/java/com/prefect/office/record/management/app/facade/prefect/offense/impl/OffenseFacadeImpl.java (hunk 5 above)
Comment (Vince-1221) [reply prefect-office-record-management-system/src/main/java/com/prefect/office/record/management/app/facade/prefect/offense/impl/OffenseFacadeImpl.java:15]: sir import java.util.List; is necessary po because of the code 

public List<Offense> getAllOffenses() {
        return offenseDao.getAllOffenses();
    }
